tensorflow/compiler/mlir/hlo/tools/mlir-hlo-opt/mlir-hlo-opt.cpp

```diff
@@ -26,6 +27,7 @@ int main(int argc, char **argv) {
   mlir::registerAllPasses();
   mlir::mhlo::registerAllMhloPasses();
   mlir::lmhlo::registerAllLmhloPasses();
+  mlir::registerAllTransformPasses();
```

Comment:
shouldn't `mlir::registerAllTransformPasses()` be just `mlir::registerTransformPasses();` and be called inside `mlir::registerAllPasses`? See `registerAllPasses()` in `mlir/include/mlir/InitAllPasses.h` .

Reply:
Okey, we will try to register the pass there.